compiler/rustc_ast/src/expand/autodiff_attrs.rs

```diff
@@ -50,8 +50,16 @@ pub enum DiffActivity {
     /// with it.
     Dual,
     /// Forward Mode, Compute derivatives for this input/output and *overwrite* the shadow argument
+    /// with it. It expects the shadow argument to be `width` times larger than the original
+    /// input/output.
+    Dualv,
```

Comment:
maybe make the `v`-ness a bool field of `Dual` instead of adding more variants?

Reply:
doesn't have to be in this PR

Reply:
sounds good, then I'll keep it for the last PR which handles reverse mode batching.